### DISCUSSION TITLE
Esperienze con valvole termostatiche e z2m

### DISCUSSION THREAD

**Discussion by FabioEight** [indomus/forum#750] [Hardware & Firmware]
Buongiorno a tutti,

Sono in cerca di valvole termostatiche Zigbee, da utilizzare con Zigbee2Mqtt (Conbee 2), quindi senza hub radio/ethernet.
Ho visto che ci sono svariati dispositivi nella lista dei devices compatibili z2m, ma volevo chiedervi se avete esperienze o suggerimenti sulla valvola che andrò ad acquistare.

Grazie!

**Comment by snakuzzo** [comment]
Per favore, usa la funzione ricerca prima di aprire nuovi topic.
Grazie
